### PR TITLE
Use glob in hunt_subtitle

### PR DESCRIPTION
**Pull request by theychx**
:fireworks:

### CODE REVIEW

File: catt/util.py [hunk 1]
```diff
@@ -36,13 +36,13 @@ def guess_mime(path):
 def hunt_subtitle(video):
     """Searches for subtitles in the current folder"""
 
-    video_path = Path(video)
-    video_path_stem_lower = video_path.stem.lower()
-    for entry_path in video_path.parent.iterdir():
-        if entry_path.is_dir():
+    video_dir_path = Path(video).parent
+    for ext in ["*.vtt", "*.VTT", "*.srt", "*.SRT"]:
+        try:
```
Comment (skorokithakis) [inline catt/util.py]: Is this not the same as:

```
for ext in ["vtt", "VTT", "srt", "SRT"]:
    subs = video_dir_path.glob("*." + ext)
    if not subs:
        continue
    return str(subs[0].resolve())
```

Comment (skorokithakis) [reply catt/util.py]: I wish globs supported multiple extensions a la regexps, or at least case-insensitivity, but I didn't find anything.

Comment (theychx) [reply catt/util.py]: > or at least case-insensitivity

I found this a bit annoying as well.

Comment (theychx) [reply catt/util.py]: ```Path.glob``` returns a generator, thus the try/except clause.

Comment (skorokithakis) [reply catt/util.py]: Err, yes, sorry, I'm missing a `list()` on my code, but I guess that would be less performant. This looks good, it just bugs me a bit that we're repeating the `*.` for each extension.

Comment (theychx) [reply catt/util.py]: I already fixed that dude. Happy new year!

Comment (skorokithakis) [reply catt/util.py]: Oh oops, so you did. Happy new year!

**Pull request merged**